copper-server-impl/src/main/java/com/pogeyan/cmis/impl/services/CmisDiscoveryService.java

```diff
@@ -60,16 +60,23 @@
 	public static class Impl {
 		public static ObjectList getContentChanges(String repositoryId, Holder<String> changeLogToken,
 				Boolean includeProperties, String filter, String orderBy, Boolean includePolicyIds, Boolean includeAcl,
-				BigInteger maxItems, ObjectInfoHandler objectInfos, IUserObject userObject, String tracingId, ISpan parentSpan) {
+				BigInteger maxItems, ObjectInfoHandler objectInfos, IUserObject userObject, String tracingId,
+				ISpan parentSpan) {
 			ISpan span = TracingApiServiceFactory.getApiService().startSpan(tracingId, parentSpan,
 					"CmisDiscoveryService::getContentChanges", null);
+			Map<String, Object> attrMap = new HashMap<String, Object>();
 			MDiscoveryServiceDAO discoveryObjectMorphiaDAO = DatabaseServiceFactory.getInstance(repositoryId)
 					.getObjectService(repositoryId, MDiscoveryServiceDAO.class);
 			MTypeManagerDAO typeManagerDAO = DatabaseServiceFactory.getInstance(repositoryId)
 					.getObjectService(repositoryId, MTypeManagerDAO.class);
 			int maxItemsInt = maxItems == null ? 10 : maxItems.intValue();
 			if (changeLogToken == null || changeLogToken.getValue() == null) {
-				throw new CmisInvalidArgumentException("change log token value should not be null!");
+				attrMap.put("error", "change log token value should not be null!, TraceId: " + span.getTraceId());
+				TracingApiServiceFactory.getApiService().updateSpan(span, true,
```

Comment:
We are setting the same text value in the attrMap and the function in updateSpan?